HW2/P5/P5.txt

```diff
@@ -0,0 +1,23 @@
+**** Nicolas Drizard helped me solve double collisions by suggesting to collide only if the grid entry was smaller or bigger than i, I choose bigger****
+
+
+This solution allowed also to avoid double locking since i>Grid[(x+xgrid),(y+ygrid)]. This solution uses the same technique in a way as the one mentioned in P2 of the homework (suggested by Isadora Nun).
+
+
+Performance:
+
+•Multithreading 1 Thread: ~12 sf/s
+•Multithreading 4 Thread: ~18 sf/s
+
+•Spatial decomposition 1 Thread: ~500 sf/s
+•Spatial decomposition 4 Thread: ~800 sf/s
```

Comment:
I got more drastic change in performance, from 13 fps to 1600 fps for serial, and 29 fps to 2000 fps for 4 threads.